.editorconfig-checker.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Format": "github-actions",
+    "Exclude": [
+        "osdep/dirent-win.h"
```

Comment:
Could we exclude it directly in .editorconfig instead?

Reply:
yeah, think that is better.